src/components/Compare/columns.ts

```diff
@@ -13,6 +13,10 @@ const CCVI_COLUMN_ID = 100;
 /**
  * Represents a column in the compare table (e.g. for the case density metric or CCVI).
  * Implementations must define the name, getValue() function, etc. as appropriate.
+ *
+ * TODO(michael): Refactor this to have a `cellContent(row: SummaryForCompare)` method that
+ * returns the rendered React component and then we can probably drop textAlign, fontSize,
+ * getFormattedValue(), getIconColor().
```

Comment:
I might take a stab at this refactor later today, since I think it'd be cleaner, but I wanted to get this PR sent out so we can get it into the preview link.